lib/fs.js

```diff
@@ -511,8 +479,8 @@ function ensureWriteStreamSync(path, options) {
   });
 });
 
-exports.access = accessAsync;
-exports.accessSync = accessSync;
+exports.access = Promise.promisify(fs.access);
```

Comment:
is `Promise.promisify(fs.access)` equivalent to `fsPromises.access`?

Reply:
There are two differences between the `util.promisify()` and `Bluebird.promisify()` functions.
One is whether the returned promise is a native promise or a bluebird.
The other is that the customization method such as this value and `Promise.[[PromiseResult]]` is different.
`fs.access()` has no this value and [[PromiseResult]] is `undefined`, so there is no difference other than returning Bluebird.

Reply:
Meaning `util.promisify(fs.access)` is equivalent to `fsPromises.access`?

Reply:
There is almost no difference between [`fs.access()`](https://github.com/nodejs/node/blob/master/lib/fs.js#L180) and [`fs.promises.access()`](https://github.com/nodejs/node/blob/master/lib/internal/fs/promises.js#L190) at least in JavaScript implementation (only the line that calls the internal API is different, the implementation of that API is C++).